app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -89,7 +114,7 @@ fun SettingGroupLabel(text: String, modifier: Modifier = Modifier) {
         text = text,
         modifier = modifier.semantics { heading() },
         color = MaterialTheme.colorScheme.primary,
-        style = MaterialTheme.typography.labelLarge,
+        style = MaterialTheme.typography.labelLarge
```

Comment:
Oh, looks like I've accidentally run ktlint on that file. Is there any formatter Accrescent uses so that I can just use that one next time? Or is it just the inbuilt from Android Studio?

Reply:
Just the inbuilt one from Android Studio: https://github.com/accrescent/accrescent/blob/master/CONTRIBUTING.md#code-style. Ideally that would be automated in CI, but I haven't looked into it yet.